qa/rpc-tests/test_framework/util.py

```diff
@@ -333,11 +338,12 @@ def _rpchost_to_args(rpchost):
         rv += ['-rpcport=' + rpcport]
     return rv
 
-def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=None, chain='elementsregtest'):
+def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=None, chain='elementsregtest', cookie_auth=False):
```

Comment:
If generalized to not only be for bitcoin parents, this variable would be unnecessary since it would just be only for `n==3`.

Reply:
I think you mean n==1

Reply:
It's n==3 that needs to set `--mainchaincookiestuff`, no?